tests/e2e_tests/health_check_tests.rs

```diff
@@ -0,0 +1,26 @@
+use crate::utils::testing::get_test_port;
+
+pub async fn test_health_check(test_type: &str) -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Can we pass the server URL in instead of test type here, since we compute that URL in the caller?